go/vt/sqlparser/parse_test.go

```diff
@@ -2140,9 +2157,7 @@ func TestIntoOutfileS3(t *testing.T) {
 			continue
 		}
 		out := String(tree)
-		if out != tcase.output {
-			t.Errorf("out: %s, want %s", out, tcase.output)
-		}
+		assert.Equal(t, tcase.output, out)
```

Comment:
Curios why using `gotest.tools/assert` where this test already uses `"github.com/stretchr/testify/require"` and can use `""github.com/stretchr/testify/assert"`. Shall we converge on `github.com/stretchr/testify`?

Reply:
Good catch!! Thank you, I missed that. I have changed it now, will be visible in the next commit.